lib/streamlit/commands/navigation.py

```diff
@@ -102,6 +103,11 @@ def navigation(
         If there is only one page in ``pages``, the navigation will be hidden
         for any value of ``position``.
 
+    expanded: bool
+        Whether the navigation menu should be expanded by default. If ``True``,
+        the navigation menu will be expanded by default. If ``False``, the
+        navigation menu will be collapsed
+
```

Comment:
IMO this docstring makes it sound like it only handles initial state (default) vs. forcing all pages to always be rendered w/o collapse option.